bridgetown-core/lib/bridgetown-core/configurations/syntax-theme.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# TODO: handle if css vs scss
```

Comment:
I could do with some guidance on how to detect and handle css vs scss here.

Reply:
SCSS is a superset of CSS, right? So could we just use plain old CSS all the time?